tests/scripts/build-release.sh

```diff
@@ -18,12 +18,12 @@ if [[ ${BRANCH_NAME} =~ release- ]]; then
     git status &
     git diff &
     build/run make -C build/release publish BRANCH_NAME=${BRANCH_NAME} TAG_WITH_SUFFIX=true AWS_ACCESS_KEY_ID=${AWS_USR} AWS_SECRET_ACCESS_KEY=${AWS_PSW} GIT_API_TOKEN=${GIT_API_TOKEN}
-elif [[ ${BRANCH_NAME} =~ master|tag ]]; then
+elif [[ ${BRANCH_NAME} =~ master ]]; then
```

Comment:
Tagged builds would need to run the `promote` command like master. If we remove the condition here, would both master and tags run it?
```suggestion
else
```

Reply:
it was my mistake in the first place, this check the branch name is master or release, checking branch name==tag is not needed, make file will take care of tags, I believe.

Reply:
Why use `=~` to check `master` and not just `==`? I'm not aware of any `master*` patterns.

Reply:
right, changed.